DESCRIPTION

```diff
@@ -1,7 +1,15 @@
 Package: genefilter
 Title: genefilter: methods for filtering genes from high-throughput experiments
 Version: 1.81.0
-Author: Robert Gentleman, Vincent J. Carey, Wolfgang Huber, Florian Hahne
+Authors@R: c(
+    person("Robert", "entleman", role = "aut"),
+    person("Vincent J.", "Carey", role = "aut"),
+     person("Wolfgang", "Huber", role = "aut"),
+      person("Florian", "Hahne", role = "aut"),
+    person("Khadijah", "Amusat", role = "ctb",
+        comment = "Converted genefilter vignette from Sweave to RMarkdown / HTML."),
+    person("Bioconductor Package Maintainer",
+        email = "maintainer@bioconductor.org", role = "cre"))
```

Comment:
- [ ] Please fix the indents. Lines 5-9, 11 should have the same indent as the existing document, which looks like a tab.
- [ ] The comment and email lines should be indented more.
- [ ] Correct spelling of Robert Gentleman's name